### PR TITLE
Display Age and Gender in CreateRelationshipForm

### PR DESCRIPTION
**Pull request by karthik-sista**
### Jira Story

- [Display Age and Gender in CreateRelationshipForm](https://osi-cwds.atlassian.net/secure/RapidBoard.jspa?rapidView=180&projectKey=RL&modal=detail&selectedIssue=RL-189)

## Tests
- [x] I have included unit tests 
- [ ] I have included feature tests 
- [ ] I have included other tests 
- [ ] I have NOT included tests 
The dataFormat prop is tested in BootstrapTable.

## Types of changes

- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [x] I have ran all tests for the project.
- [x] I have ran lint/rubocop check for the changed/new files.
- [x] My code is in a stable state ready to be deployable, but not necessarily complete.
- [x] I promise on my honor as a CWDS developer to ensure I don't break the build, to check Lint/Rubocop, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

**Comment by wmitchell-cambria** [comment]
ok to test

**Comment by wmitchell-cambria** [comment]
retest this please

**Comment by wmitchell-cambria** [comment]
ok to test

**Comment by karthik-sista** [comment]
retest this please

File: app/javascript/views/ScreeningCreateRelationship.jsx [hunk 1]
```diff
@@ -103,8 +116,5 @@ export default class ScreeningCreateRelationship extends React.Component {
 }
 
 ScreeningCreateRelationship.propTypes = {
-  data: PropTypes.arrayOf(PropTypes.shape({
-    focus_person: PropTypes.string,
-    related_person: PropTypes.string,
-  })),
+  data: PropTypes.array,
```
Comment (wmitchell-cambria) [inline app/javascript/views/ScreeningCreateRelationship.jsx]: Heads up that we are weakening the propTypes here.

File: app/javascript/views/ScreeningCreateRelationship.jsx [hunk 2]
```diff
@@ -27,17 +28,29 @@ export default class ScreeningCreateRelationship extends React.Component {
...
+  displayFormatter(cell) {
+    return (
+      <div>
+        <ul className='unstyled-list'>
+          <li>{cell.name}</li>
+          <li>{cell.age}</li>
+          <li>{GENDERS_LEGACY[cell.gender]}</li>
+        </ul>
+      </div>
+    )
```
Comment (NagaPradeepGajula-osi) [inline app/javascript/views/ScreeningCreateRelationship.jsx]: Are we testing this list order somewhere already? If not can we add it in this PR? Though you have a expectation updated to check in string format.

File: spec/javascripts/views/ScreeningCreateRelationshipSpec.jsx [hunk 3]
```diff
@@ -36,4 +37,13 @@ describe('ScreeningCreateRelationship', () => {
     cancel.props.onClick()
     expect(wrapper.state().show).toEqual(false)
   })
+
+  it('displayFormatter has name, age and gender', () => {
```
Comment (NagaPradeepGajula-osi) [inline spec/javascripts/views/ScreeningCreateRelationshipSpec.jsx]: Awesome. this looks good.

**Pull request merged**